### PR TITLE
chore: Support subgraph healtch check after upgrade

### PR DESCRIPTION
**Pull request by chefjackson**
---

## PR-Codex overview
This PR updates subgraph configurations in various components and hooks for Trading Competition, Pottery, NFT Market, and Limit Orders. 

### Detailed summary
- Updated subgraph configurations in Trading Competition components and hooks
- Refactored subgraph settings in Pottery and NFT Market components
- Adjusted subgraph configurations in Limit Orders components

> The following files were skipped due to too many changes: `apps/web/src/components/SubgraphHealthIndicator/FloatingSubgraphHealthIndicators.tsx`, `apps/web/src/hooks/useSubgraphHealth.ts`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: apps/web/src/views/TradingCompetition/mobox/components/YourScore/MoboxYourScore.tsx [hunk 1]
```diff
@@ -47,7 +50,7 @@ const MoboxYourScore: React.FC<React.PropsWithChildren<YourScoreProps>> = ({
         </RibbonWithImage>
       )}
       <ScoreCard
-        subgraphName="pancakeswap/trading-competition-v3"
+        subgraph={TC_MOBOX_SUBGRAPH}
```
Comment (chef-jojo) [inline apps/web/src/views/TradingCompetition/mobox/components/YourScore/MoboxYourScore.tsx:53]: thinking maybe we should consider removing those old TC 😅

Comment (chefjackson) [reply apps/web/src/views/TradingCompetition/mobox/components/YourScore/MoboxYourScore.tsx:53]: what does TC mean actually? 😅 I'm blindly using the old name here

**Review by chef-jojo — APPROVED** [review]

File: apps/web/src/views/TradingCompetition/mobox/components/YourScore/MoboxYourScore.tsx (hunk 1 above)
Comment (chefjackson) [reply apps/web/src/views/TradingCompetition/mobox/components/YourScore/MoboxYourScore.tsx:53]: oh right is it abbreviation of trading competition?

Comment (chef-jojo) [reply apps/web/src/views/TradingCompetition/mobox/components/YourScore/MoboxYourScore.tsx:53]: Trading Competition lol. Just keep hosting it for claiming before

**Pull request merged**